src/xeus_calc_interpreter.cpp

```diff
@@ -0,0 +1,178 @@
+#include <iostream>
+#include "xeus_calc_interpreter.hpp"
+#include <vector>
+#include <sstream>
+#include <stack>
+#include "xeus/xinterpreter.hpp"
```

Comment:
Nitpicking: STL headers first, then headers from dependencies and finally headers from the library.